its/ruling/src/integrationTest/resources/expected/kubernetes/kubernetes-S5849.json

```diff
@@ -6,9 +6,9 @@
 11
 ],
 "kubernetes-project:ruling/src/integrationTest/resources/sources/kubernetes/helm/corner-cases/templates/multiple-documents.yaml": [
+0,
 11,
-27,
-44
```

Comment:
I guess this will be fixed back after implementing https://sonarsource.atlassian.net/browse/SONARIAC-1268

Reply:
Could you please clarify? Your PR removes blank lines at the end of file, so why is that ticket relevant?

Reply:
Let me check...